applications/launchpad/docker_rig/tor.Dockerfile

```
@@ -1,23 +1,52 @@
 # Package build
-FROM alpine:latest
 
+ARG ALPINE_VERSION=3.16
+
+FROM alpine:$ALPINE_VERSION
+
+ARG ALPINE_VERSION
 ARG BUILDPLATFORM
 ARG VERSION=1.0.1
 
 # https://pkgs.alpinelinux.org/packages?name=tor&branch=v3.16&repo=community
-ARG TOR_VERSION=0.4.7.7-r1
+ARG TOR_VERSION=0.4.7.8-r0
 
+# Install tor with a minimum version
 RUN apk update \
  && apk upgrade \
- && apk add tor=$TOR_VERSION \
+ && apk add grep curl tor>$TOR_VERSION \
  && rm /var/cache/apk/*
 
-EXPOSE 9050
-EXPOSE 9051
 ENV dockerfile_version=$VERSION
 ENV dockerfile_build_arch=$BUILDPLATFORM
+ENV apline_verison=$ALPINE_VERSION
```

```suggestion
ENV alpine_version=$ALPINE_VERSION
```
is this used inside the tor container?

---

Not currently used inside hosted docker image. Good catch. Sure when I used it would have wasted too many cycles to finding the typo.